Makefile

```diff
@@ -99,6 +99,25 @@ run-notebook-tests:
 	$(MAKE) configure-vault
 	$(MAKE) -C manager run-notebook-tests
 
+.PHONY: run-integration-tests-no-cluster-scope
+run-integration-tests-no-cluster-scope: export DOCKER_HOSTNAME?=localhost:5000
+run-integration-tests-no-cluster-scope: export DOCKER_NAMESPACE?=fybrik-system
+run-integration-tests-no-cluster-scope: export VALUES_FILE=charts/fybrik/integration-tests.no-cluster-scope.values.yaml
+run-integration-tests-no-cluster-scope:
+	$(MAKE) kind
+	$(MAKE) cluster-prepare
+	$(MAKE) docker-build docker-push
+	$(MAKE) -C test/services docker-build docker-push
+	$(MAKE) cluster-prepare-wait
+	$(MAKE) -C charts test
+	$(MAKE) deploy
+	$(MAKE) configure-vault
+	$(MAKE) -C modules helm
+	$(MAKE) -C modules helm-uninstall # Uninstalls the deployed tests from previous command
+	$(MAKE) -C pkg/helm test
+	$(MAKE) -C manager run-integration-tests
+	$(MAKE) -C modules test
```

Comment:
Not needed.

Reply:
I see that the `test` target in the modules/Makefile is empty and has the following comment 
"# Tests if implicit copy definitions can be installed"

Do we need it? When do we use it?

Reply:
I think it's leftover from the time we had the implicit copy modules in fybrik repo. As far as I know we can remove it.

Reply:
so, please remove it

Reply:
Done. Thanks